docs/temp-mail/temp-mail-vs-adguard-mail.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Differences between Temp Mail in AdGuard Mail and AdGuard Temp Mail
+sidebar_position: 2
+---
+
+AdGuard offers two temporary email services:
+
+- [AdGuard Temp Mail](https://adguard.com/adguard-temp-mail/overview.html) — a disposable address generator
+
+- [AdGuard Mail](https://adguard-mail.com/welcome.html) — an email management product with Temp Mail functionality
+
+Both services are suitable for one-time tasks like activating a free trial or getting promo codes. Here’s a comparison to help you choose the one that best fits your needs.
+
+| **Feature** | **AdGuard Temp Mail** | **Temp Mail in AdGuard Mail** |
+|-----------------|---------------------------------|------------------------------------------|
+| **Lifetime**  | 7 days or until the page is closed  | No limit |
+| **Availability**  | Only on the webpage you open it on | Web version, mobile and desktop  |
+| **Registration**  |  Not required | Account required |
+
+## Lifetime of addresses
+
+In AdGuard Temp Mail, the email address is deleted after 7 days or when you close the website. With AdGuard Mail, the temporary address lasts as long as you want.
+
+## Platform availability
+
+In AdGuard Temp Mail, the generated email address is only available on the page where you open it, you can’t use it on any other device. Once you close the page or clear your browser cache, the address and its inbox will be deleted.
+
+With AdGuard Mail, you can use your temporary email in the web version, desktop program, or mobile application. You can access your inbox from any device at any time.
+
+## Registration
+
+AdGuard Temp Mail doesn’t require registration. Just visit the website, pass the CAPTCHA, and a disposable email address will be generated.
+
+In AdGuard Mail, an account is required. Since it’s part of the AdGuard product suite, you can log in using your [AdGuard account](https://auth.adguardaccount.com/login.html).
```

Comment:
```suggestion
AdGuard Mail requires an account. Since it’s part of the AdGuard product suite, you can log in using your [AdGuard account](https://auth.adguardaccount.com/login.html).
```